LAB12: Added Week 13 Lab 12 to the Website

rvarde2 opened the pull request:
Lab on Introduction to Wireshark

rvarde2 committed:
LAB12: Added Week 13 Lab 12 to the Website

Lab on Introduction to Wireshark

kaytwo reviewed (changes requested):
This looks really good! Can you make two structural changes:

For any link that isn't going to change based on which semester the class is taught (i.e. the wireshark download link), you can just use markdown syntax to put the link directly in the content of the page.

For the various python files, I believe it would be better to just directly embed them as source code in the document itself, then students can copy and paste them into their editor rather than figuring out how to download the file into their environment (or, more likely, downloading, opening, and then copy pasting it in anyway). There might be some issues with spacing if they don't paste correctly, but learning how to paste code properly is not a bad outcome I guess 🙃.

rvarde2 committed:
LAB12: Week 13, Lab 12 Additional Changes

-- Instead of links to python scripts, code is directly embeded in page.
-- Semester independent links are directly added in the content.

rvarde2 committed:
LAB12: Highlighted Sentence showing specific file is important for quiz

moebit committed:
minor edits

moebit committed:
reverse fix
